usr.sbin/freebsd-update/freebsd-update.sh

```diff
@@ -2200,6 +2212,13 @@ fetch_run () {
 	done
 	fetch_tagsanity || return 1
 
+	# If there are no updates upstream there is no need for
+	# further processing (except forced fetch).
+	if [ $FORCEFETCH -eq 0 ]; then
+		fetch_compare_new_tag_to_current_tag &&
```

Comment:
What happens if `tag` does not exist? (e.g. a user has removed all the contents of `workdir`). `fetch_compare_new_tag_to_current_tag` returns `0`, and no update will be required?

Will it always leave a `tag.new` file?

Reply:
fetch_compare_new_tag_to_current_tag does nothing if either tag or tag.new don't exist, which means it carries on with current behavior.  tag.new isn't my file, it's transient in freebsd-update, I'm utilizing it at the right time.

Reply:
Commit added to return `1` if the initial `if` fails.

Reply:
I think writing this as
```
if fetch_compare_new_tag_to_current_tag; then
    echo "No updates...."
    rm -f tag.new
    return 0
fi
```
would be more typical

Reply:
or actually
```
if [ $FORCEFETCH -eq 0 ] && fetch_compare_new...
    echo "No updates..."
```